Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -111,6 +113,7 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
     virtual void didCommitLayerTree(const RemoteLayerTreeTransaction&, const RemoteScrollingCoordinatorTransaction&) { }
 
     void asyncSetLayerContents(WebCore::GraphicsLayer::PlatformLayerID, ImageBufferBackendHandle&&);
+    void layerWillBeUsedForRemoteFrame(WebCore::GraphicsLayer::PlatformLayerID, WebCore::FrameIdentifier);
```

Comment:
I think we should talk about "hosting a frame" (like `ScrollingTreeFrameHostingNode`)